Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib

```diff
@@ -0,0 +1,72 @@
+                                    <rect key="frame" x="0.0" y="25.333333333333329" width="332.66666666666669" height="20.333333333333329"/>
+                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                    <nil key="textColor"/>
+                                    <nil key="highlightedColor"/>
+                                </label>
+                            </subviews>
+                        </stackView>
+                        <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="person.circle" catalog="system" translatesAutoresizingMaskIntoConstraints="NO" id="M9w-Ep-1bS">
+                            <rect key="frame" x="435" y="13.666666666666671" width="40" height="38.666666666666671"/>
+                            <constraints>
+                                <constraint firstAttribute="width" constant="40" id="Zg6-RA-89r"/>
+                                <constraint firstAttribute="height" constant="40" id="hZ3-73-ANY"/>
+                            </constraints>
+                        </imageView>
+                    </subviews>
+                    <constraints>
+                        <constraint firstItem="fcg-1x-5Or" firstAttribute="width" secondItem="hyi-uA-h8a" secondAttribute="width" multiplier="0.7" id="POE-tS-tsh"/>
+                        <constraint firstItem="fcg-1x-5Or" firstAttribute="leading" secondItem="hyi-uA-h8a" secondAttribute="leading" id="Xmg-9w-RzD"/>
+                    </constraints>
+                </stackView>
+            </subviews>
+            <viewLayoutGuide key="safeArea" id="vUN-kp-3ea"/>
+            <constraints>
+                <constraint firstItem="hyi-uA-h8a" firstAttribute="top" secondItem="iN0-l3-epB" secondAttribute="top" constant="64" id="Pf3-3S-j9j"/>
+                <constraint firstItem="hyi-uA-h8a" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" constant="30" id="RPd-g5-srf"/>
+                <constraint firstAttribute="bottom" secondItem="hyi-uA-h8a" secondAttribute="bottom" constant="20" id="dk5-jT-eBi"/>
+                <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="hyi-uA-h8a" secondAttribute="trailing" constant="30" id="sbK-8I-GiQ"/>
+            </constraints>
+            <freeformSimulatedSizeMetrics key="simulatedDestinationMetrics"/>
+            <point key="canvasLocation" x="248.09160305343511" y="-19.366197183098592"/>
+        </view>
+    </objects>
+    <resources>
+        <image name="person.circle" catalog="system" width="128" height="123"/>
+    </resources>
+</document>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21507" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
    <device id="retina6_12" orientation="portrait" appearance="light"/>
    <dependencies>
        <deployment identifier="iOS"/>
        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21505"/>
        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
    </dependencies>
    <objects>
        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="HomeHeaderView" customModule="Healthy" customModuleProvider="target">
            <connections>
                <outlet property="contentView" destination="iN0-l3-epB" id="7FY-c0-NXm"/>
                <outlet property="subtitleLabel" destination="anc-yl-iPe" id="k79-72-cnN"/>
                <outlet property="titleLabel" destination="DRb-b2-9Sw" id="DXl-go-1Hb"/>
                <outlet property="userImageView" destination="M9w-Ep-1bS" id="O2L-d6-vEe"/>
            </connections>
        </placeholder>
        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
        <view contentMode="scaleToFill" id="iN0-l3-epB">
            <rect key="frame" x="0.0" y="0.0" width="535" height="149"/>
            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
            <subviews>
                <stackView opaque="NO" contentMode="scaleToFill" distribution="equalCentering" alignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="hyi-uA-h8a">
                    <rect key="frame" x="20" y="20" width="495" height="109"/>
                    <subviews>
                        <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="5" translatesAutoresizingMaskIntoConstraints="NO" id="fcg-1x-5Or">
                            <rect key="frame" x="0.0" y="31.666666666666668" width="56.666666666666664" height="45.666666666666657"/>
                            <subviews>
                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="title" textAlignment="natural" lineBreakMode="wordWrap" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="DRb-b2-9Sw">
                                    <rect key="frame" x="0.0" y="0.0" width="56.666666666666664" height="20.333333333333332"/>
                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                    <nil key="textColor"/>
                                    <nil key="highlightedColor"/>
                                </label>
                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="subtitle" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" minimumScaleFactor="0.5" translatesAutoresizingMaskIntoConstraints="NO" id="anc-yl-iPe">
                                    <rect key="frame" x="0.0" y="25.333333333333336" width="56.666666666666664" height="20.333333333333336"/>
                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                    <nil key="textColor"/>
                                    <nil key="highlightedColor"/>
                                </label>
                            </subviews>
                        </stackView>
                        <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="person.circle" catalog="system" translatesAutoresizingMaskIntoConstraints="NO" id="M9w-Ep-1bS">
                            <rect key="frame" x="455" y="35.666666666666664" width="40" height="38.666666666666664"/>
                            <constraints>
                                <constraint firstAttribute="width" constant="40" id="Zg6-RA-89r"/>
                                <constraint firstAttribute="height" constant="40" id="hZ3-73-ANY"/>
                            </constraints>
                        </imageView>
                    </subviews>
                </stackView>
            </subviews>
            <constraints>
                <constraint firstItem="hyi-uA-h8a" firstAttribute="top" secondItem="iN0-l3-epB" secondAttribute="top" constant="20" id="Pf3-3S-j9j"/>
                <constraint firstItem="hyi-uA-h8a" firstAttribute="leading" secondItem="iN0-l3-epB" secondAttribute="leading" constant="20" id="RPd-g5-srf"/>
                <constraint firstAttribute="bottom" secondItem="hyi-uA-h8a" secondAttribute="bottom" constant="20" id="dk5-jT-eBi"/>
                <constraint firstAttribute="trailing" secondItem="hyi-uA-h8a" secondAttribute="trailing" constant="20" id="sbK-8I-GiQ"/>
            </constraints>
            <freeformSimulatedSizeMetrics key="simulatedDestinationMetrics"/>
            <point key="canvasLocation" x="248.09160305343511" y="-19.366197183098592"/>
        </view>
    </objects>
    <resources>
        <image name="person.circle" catalog="system" width="128" height="123"/>
    </resources>
</document>
```

I've made some changes to reflect the suggestions, please feel free to add and discuss any point if needed. 

Steps: 
- Open XIB file as source code 
- Replace the content

Reply:
Diffs for reference 
```
diff --git a/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.swift b/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.swift
index bd0b1d1..66d8e72 100644
--- a/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.swift
+++ b/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.swift
@@ -29,8 +29,13 @@ final class HomeHeaderView: UIView {
     }
     
     private func configureLayout() {
+        titleLabel.applyStyle(.signupTitle)
+        
+        // TODO: - Waiting for add home subtitle label style.
+        subtitleLabel.applyStyle(.signupSubtitle)
+        
         NSLayoutConstraint.activate([
-            contentView.heightAnchor.constraint(equalToConstant: 120),
+            contentView.heightAnchor.constraint(equalToConstant: 128),
             contentView.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width)
         ])
     }
@@ -39,7 +44,7 @@ final class HomeHeaderView: UIView {
         titleLabel.text = viewModel.title
         subtitleLabel.text = viewModel.subtitle
         
-        // TODO: - Waiting when add kingfisher (or any library for handle image cashing)
+        // TODO: - Waiting for adding kingfisher (or any library for handle image cashing).
     }
 }
 
diff --git a/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib b/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib
index 93a8095..237ecd5 100644
--- a/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib
+++ b/Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib
@@ -5,7 +5,6 @@
         <deployment identifier="iOS"/>
         <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21505"/>
         <capability name="Safe area layout guides" minToolsVersion="9.0"/>
-        <capability name="System colors in document resources" minToolsVersion="11.0"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <objects>
@@ -19,53 +18,55 @@
         </placeholder>
         <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
         <view contentMode="scaleToFill" id="iN0-l3-epB">
-            <rect key="frame" x="0.0" y="0.0" width="535" height="109"/>
+            <rect key="frame" x="0.0" y="0.0" width="535" height="149"/>
             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
             <subviews>
-                <stackView opaque="NO" contentMode="scaleToFill" alignment="center" spacing="208" translatesAutoresizingMaskIntoConstraints="NO" id="hyi-uA-h8a">
-                    <rect key="frame" x="30" y="44.333333333333343" width="475" height="45.666666666666657"/>
+                <stackView opaque="NO" contentMode="scaleToFill" distribution="equalCentering" alignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="hyi-uA-h8a">
+                    <rect key="frame" x="30" y="64" width="475" height="65"/>
                     <subviews>
                         <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="5" translatesAutoresizingMaskIntoConstraints="NO" id="fcg-1x-5Or">
-                            <rect key="frame" x="0.0" y="0.0" width="227" height="45.666666666666664"/>
+                            <rect key="frame" x="0.0" y="9.6666666666666679" width="332.66666666666669" height="45.666666666666657"/>
                             <subviews>
-                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="title" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="DRb-b2-9Sw">
-                                    <rect key="frame" x="0.0" y="0.0" width="227" height="20.333333333333332"/>
+                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="title" textAlignment="natural" lineBreakMode="wordWrap" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="DRb-b2-9Sw">
+                                    <rect key="frame" x="0.0" y="0.0" width="332.66666666666669" height="20.333333333333332"/>
                                     <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                     <nil key="textColor"/>
                                     <nil key="highlightedColor"/>
                                 </label>
-                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="subtitle" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="anc-yl-iPe">
-                                    <rect key="frame" x="0.0" y="25.333333333333336" width="227" height="20.333333333333336"/>
+                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="subtitle" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" minimumScaleFactor="0.5" translatesAutoresizingMaskIntoConstraints="NO" id="anc-yl-iPe">
+                                    <rect key="frame" x="0.0" y="25.333333333333329" width="332.66666666666669" height="20.333333333333329"/>
                                     <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                     <nil key="textColor"/>
                                     <nil key="highlightedColor"/>
                                 </label>
                             </subviews>
                         </stackView>
-                        <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="M9w-Ep-1bS">
-                            <rect key="frame" x="435" y="2.6666666666666643" width="40" height="40"/>
+                        <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="person.circle" catalog="system" translatesAutoresizingMaskIntoConstraints="NO" id="M9w-Ep-1bS">
+                            <rect key="frame" x="435" y="13.666666666666671" width="40" height="38.666666666666671"/>
                             <constraints>
-                                <constraint firstAttribute="height" constant="40" id="R3G-1c-Zwm"/>
-                                <constraint firstAttribute="width" constant="40" id="Rp4-XE-P4K"/>
+                                <constraint firstAttribute="width" constant="40" id="Zg6-RA-89r"/>
+                                <constraint firstAttribute="height" constant="40" id="hZ3-73-ANY"/>
                             </constraints>
                         </imageView>
                     </subviews>
+                    <constraints>
+                        <constraint firstItem="fcg-1x-5Or" firstAttribute="width" secondItem="hyi-uA-h8a" secondAttribute="width" multiplier="0.7" id="POE-tS-tsh"/>
+                        <constraint firstItem="fcg-1x-5Or" firstAttribute="leading" secondItem="hyi-uA-h8a" secondAttribute="leading" id="Xmg-9w-RzD"/>
+                    </constraints>
                 </stackView>
             </subviews>
             <viewLayoutGuide key="safeArea" id="vUN-kp-3ea"/>
-            <color key="backgroundColor" systemColor="systemBlueColor"/>
             <constraints>
-                <constraint firstItem="hyi-uA-h8a" firstAttribute="centerY" secondItem="vUN-kp-3ea" secondAttribute="centerY" id="Jgk-NY-PQa"/>
+                <constraint firstItem="hyi-uA-h8a" firstAttribute="top" secondItem="iN0-l3-epB" secondAttribute="top" constant="64" id="Pf3-3S-j9j"/>
                 <constraint firstItem="hyi-uA-h8a" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" constant="30" id="RPd-g5-srf"/>
+                <constraint firstAttribute="bottom" secondItem="hyi-uA-h8a" secondAttribute="bottom" constant="20" id="dk5-jT-eBi"/>
                 <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="hyi-uA-h8a" secondAttribute="trailing" constant="30" id="sbK-8I-GiQ"/>
             </constraints>
             <freeformSimulatedSizeMetrics key="simulatedDestinationMetrics"/>
-            <point key="canvasLocation" x="248.09160305343511" y="-33.450704225352112"/>
+            <point key="canvasLocation" x="248.09160305343511" y="-19.366197183098592"/>
         </view>
     </objects>
     <resources>
-        <systemColor name="systemBlueColor">
-            <color red="0.0" green="0.47843137254901963" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-        </systemColor>
+        <image name="person.circle" catalog="system" width="128" height="123"/>
     </resources>
 </document>
```

Reply:
All changes are understandable.